playback/src/mixer/mod.rs

```diff
@@ -17,7 +17,7 @@ pub trait Mixer: Send {
 }
 
 pub trait AudioFilter {
-    fn modify_stream(&self, data: &mut [f64]);
+    fn peek(&self) -> f64;
```

Comment:
`peek` is already a known idiom in Rust, that we might want to avoid using here with a different meaning: https://doc.rust-lang.org/stable/std/iter/struct.Peekable.html

Maybe as simple as `volume`?

Reply:
I'm fine with whatever but there is already a volume function.

Reply:
Hmm yes. That one gets the position on the volume slider. This one gets the PCM attenuation factor. Maybe we should rename them appropriately.

Reply:
How about `attenuation_factor`?

Reply:
👍